src/main/java/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -132,14 +138,15 @@ public Random rng() {
      * schema.
      *
      * @param db_type
-     * @throws SQLException
      */
     public String getDatabaseDDLPath(DatabaseType db_type) {
 
+        // The order matters!
         List<String> names = new ArrayList<>();
         if (db_type != null) {
             names.add("ddl-" + db_type.name().toLowerCase() + ".sql");
         }
+        names.add("ddl-generic.sql");
```

Comment:
Note that this is now ddl-generic to fit the theme instead of the old auctionmark-ddl, tpcc-ddl, etc.